doc/samtools-samples.1

```diff
@@ -0,0 +1,95 @@
+'\" t
```

Comment:
I've added a man page stub with the help menu taken from the code, but you should enrich it with more detailed descriptions of the command and options. Also, it would be beneficial to have the use cases from the PR notes imported here.